Sources/Commands.swift

```diff
@@ -242,7 +242,7 @@ public func command<A:ArgumentDescriptor, A1:ArgumentDescriptor, A2:ArgumentDesc
         BoxedArgumentDescriptor(value: descriptor5),
     ])
 
-    if parser.hasOption("help") {
+    if parser.has(option: "help") {
```

Comment:
I'm not certain, but I think this might be discourage from the Swift API guidelines.

https://swift.org/documentation/api-design-guidelines/#argument-labels

<img width="569" alt="screen shot 2016-12-12 at 16 59 34" src="https://cloud.githubusercontent.com/assets/44164/21108237/6fc118f4-c08c-11e6-80e5-1d711611c426.png">